functional-samples/cookbook.geolocation-offscreen/README.md

```diff
@@ -0,0 +1,17 @@
+This recipe shows how to get `geolocation` access within a service worker
+
+## Context
+
+To get geolocation information in extensions, use the same [`navigator.geolocation`][6] DOM API that any website normally would. This demo shows how to access this data in a Service Worker (via [offscreenDocument][2]. We also have samples for a [popup][3], or a [content script][4].
+
+## Running this extension
```

Comment:
I tested the offscreen docs sample. When I click on the action icon when I'm on the chrome://extensions page, nothing happens. I inspected the SW and I didn't see any logs 

Next, I tried navigating to example.com and clicking on the action icon. I inspected the service worker, but there was no log.

We should probably add more steps.